src/utils/dates.js

```diff
@@ -0,0 +1,10 @@
+export function formatIsoString(s) {
+  const options = {
+    month: 'short',
+    day: 'numeric',
+    year: 'numeric',
+    hour: '2-digit',
+    minute:'2-digit'
+  };
+  return new Date(s).toLocaleString([], options);
```

Comment:
Lets just us moment.js for formatting our timestamps.

Reply:
Done. But I also kept the utils function, because Marcus mentioned that we should have one for consistent time formatting across the program.